boards/arm/efr32_radio/Kconfig.defconfig

```diff
@@ -32,6 +32,7 @@ config LOG_BACKEND_SWO_FREQ_HZ
 if SOC_GECKO_USE_RAIL
 
 config FPU
+	default n if SOC_GECKO_SERIES1
```

Comment:
Hmmm, we don't need to disable FPU if we're not using RAIL. So we should check if we're `SOC_GECKO_SERIES1` and if we're using RAIL here IMO.

Reply:
This is already checked on line 32

Reply:
Monday mornings ;) You're correct, thanks